src/impl_methods.rs

```diff
@@ -73,6 +73,11 @@ impl<A, S, D> ArrayBase<S, D> where S: Data<Elem=A>, D: Dimension
         self.dim.clone()
     }
 
+    /// Return the strides of the array as they are stored in the array.
+    pub fn raw_strides(&self) -> D {
```

Comment:
I rolled back this change by amending the original PR. Using the `strides` field directly (I didn't think about the `linalg` module having direct access to `ArrayBase`'s private fields due to `ArrayBase` being defined in `lib` and `linalg` being a child module of `lib`).